### ISSUE TITLE
swallowing errors

### ISSUE THREAD

**Issue by boclair** [arqex/fluxify#3]
Fluxify is swallowing errors.

Even a simple example like this does not show the error in the console.  It does not display the "oops" is not defined error.  This makes developing with fluxify very difficult if no errors are displayed.

```
<!DOCTYPE html>
<html>
  <head>
    <script type="text/javascript" src="http://rawgit.com/arqex/fluxify/master/build/fluxify.js"></script>
    <script>
      var dispatcher = fluxify.dispatcher;

      dispatcher.register( function(){
        console.log( "here" );
        oops.oops();
        console.log( "not here" );
      });

      dispatcher.dispatch( "test" ); 
    </script>
  </head>

  <body>
  </body>
</html>
```

**Comment by arqex** [comment]
Ey @boclair 

You are right, promises in the dispatcher hides the error. They expect you to catch every dispatch, but I think there is a point on notifying the errors in them. I have updated fluxify and added a default error handler that shows errors in the console. Here it is the result:

http://jsbin.com/zesagogexo/1/edit?html,js,console

Please update fluxify to start a better development ;)

**Comment by frewsxcv** [comment]
Was this fixed in e1d0c1e6d2a39469693484a04aafc468df12b16a? If so, this can probably be closed
